components/DropDown/index.jsx

```diff
@@ -0,0 +1,115 @@
+import React, {PropTypes, Component} from 'react';
+
+import styles from './index.scss';
+
+import {
+  clearfix
+} from '../index.scss';
+
+export default class Dropdown extends Component {
+  static propTypes = {
+    items: PropTypes.array.isRequired,
+    position: PropTypes.oneOf(['top', 'bottom']),
+    shade: PropTypes.oneOf(['light', 'dark'])
+  };
+
+  static defaultProps = {
+    position: 'top',
+    shade: 'light'
+  }
+
+  state = {
+    isMounted: true,
+    isOpen: false,
+    selected: this.getSelection()
+  }
+
+  componentWillMount () {
+    document.addEventListener('click', this.handleDocumentClick, false);
```

Comment:
`onClickDocument` would be the standard way to name callback in handlers we agreed on.